activemq-karaf-itest/src/test/java/org/apache/activemq/karaf/itest/ActiveMQBrokerNdExternalCamelFeatureTest.java

```diff
@@ -50,7 +50,7 @@
     public void test() throws Throwable {
         assertFeatureInstalled("activemq");
         installAndAssertFeature("camel");
-        installAndAssertFeature("camel-jms");
+        installAndAssertFeature("activemq-camel");
```

Comment:
@jbonofre  - thoughts on this? this might also be a mistake

Reply:
This should NOT be reverted. activemq-camel removed in 5.17.0

Reply:
Yep (though note the test is ignored so it wont actually make any immediate difference in the end).